src/test/scala/org/kibanaLoadTest/simulation/BaseSimulation.scala

```diff
@@ -104,6 +105,8 @@ class BaseSimulation extends Simulation {
 
   after {
     SimulationHelper.copyRunConfigurationToReportPath()
+    // move response log from /target to gatling report folder
+    moveResponseLogToResultsDir
```

Comment:
This seems cool. 
I'm guessing since this is a unit type, you are "returning" void here, while also shuffling files about.

It's a like a cool trick! lol

Do I have that correct?

Reply:
You got it right :)